components/ServicesTechnology/ServicesTechnologyCard/ServicesTechnologyCard.tsx

```diff
@@ -0,0 +1,25 @@
+import { useRouter } from 'next/router';
+
+export default function ServicesTechnologyCard(
+  {
+    title,
+    description,
+    link,
+  }:
+  {
+    title: string;
+    description: string;
+    link: string
+  },
+) {
+  const { pathname } = useRouter();
+
+  return (
+    <a href={link} className={`services-technology-card services-technology-card--${pathname.slice(1)}`}>
```

Comment:
I think these cards will be internal links and it would be better to use the Link component from nextjs. 

Unfortunately, this topic is not mentioned in the task.

Reply:
Yes, that's what I thought, but I'm having problems using Link, for some reason, it's not displayed in the DOM tree.